dataprofiler/profilers/datetime_column_profile.py

```diff
@@ -127,6 +127,33 @@ def data_type_ratio(self):
             return float(self.match_count) / self.sample_size
         return None
 
+    def diff(self, other_profile, options=None):
+        """
+        Generates the differences between max, min, and formats of two DateTime columns
+
+        :return: Dict containing the differences between max, min, and format in their
+        appropriate output formats
+        :rtype: dict
+        """
+        cls = self.__class__
+        if not isinstance(other_profile, cls):
+            raise TypeError("Unsupported operand type(s) for diff: '{}' "
+                            "and '{}'".format(cls.__name__,
+                                              other_profile.__class__.__name__))
+
+        # We can use find_diff_of_numbers for max/min to get a timedelta object,
+        # since datetime objects can be compared and subtracted naturally
+        min_diff = utils.find_diff_of_numbers(self._dt_obj_min, other_profile._dt_obj_min)
+        max_diff = utils.find_diff_of_numbers(self._dt_obj_max, other_profile._dt_obj_max)
+
+        differences = {
+            "min": utils.timedelta_to_readable_output(min_diff),
```

Comment:
if we update for the new function as described below, can then just call it here and don't need the diff funcs above.